hosting/static-website/README.md

```diff
@@ -0,0 +1,128 @@
+
+- git
+- DFX 
+
+git can be installed from various package managers. DFX can be installed following the instructions [here](https://smartcontracts.org/docs/quickstart/local-quickstart.html#download-and-install).
+
+### Install
+Install the example dapp project:
+
+```bash
+$ git clone https://github.com/dfinity/examples
+$ cd motoko/static-website
+```
+
+## Documentation
+There are two parts in this dapp. One is the website and the other is `dfx.json`. Since there's no backend, there is not any benefit of using the `dfx new project_name` command to set up a template. The `dfx.json` file is all that is needed.
+
+### Website
+The website is really simple. It consists of an HTML file, a CSS file and a PNG file. The content of the HTML file looks like this:
+
+```html
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Static Website</title>
+        <base href="/">
+        <link type="text/css" rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+        <img src="logo.png" alt="DFINITY logo" />
+    </body>
+</html>
+```
+
+The only styling done in the CSS file is aligning the logo image:
+
+```css
+img {
+    max-width: 50vw;
+    max-height: 25vw;
+    display: block;
+    margin: auto;
+}
+```
+
+The top level folder is called `assets` and inside this folder, the CSS file and image is stored in a folder called `assets` and the HTML file `index.html` is stored in a folder called `src`. 
+
+### dfx.json
+The `dfx.json` file is a configuration file which specifies the canister used for the dapp. In this case only one canister is needed, and besides the canister configuration, `dfx.json` also includes information about DFX version, build settings and network settings.
+
+```json
+{
+    "canisters": {
+        "www": {
+            "frontend": {
+                "entrypoint": "assets/src/index.html"
+            },
+            "source": [
+                "assets/assets",
+                "assets/src"
+            ],
+            "type": "assets"
+        }
+    },
+    "defaults": {
+        "build": {
+            "args": "",
+            "packtool": ""
+        }
+    },
+    "dfx": "0.8.4",
+    "networks": {
+        "local": {
+            "bind": "127.0.0.1:8000",
+            "type": "ephemeral"
+        }
+    },
+    "version": 1
+}
+```
+
+This is all needed for creating a canister smart contract for hosting a static website on the IC.
+
+## Deployment
+The local network is started by running this command:
+
+```bash
+$ dfx start --background
+```
+
+When the local network is up and running, run this command to deploy the canisters:
+
+```bash
+$ dfx deploy
+```
+
+
+## Testing
+The URL for the frontend depends on the canister ID.  Local canister IDs can be obtained using `dfx canister id`, in this case `dfx canister id www`. When deployed, the URL will look like this:
+
+**https://<ui_canister_id>.localhost:8000**
```

Comment:
I wonder if we should note: this works for Chrome, Firefox, Opera, and I'm sure more.  Unfortunately it doesn't for Safari, which would require a URL like `http://localhost:8000?canisterId=<ui_canister_id>`

Also, for local development, only `http://` will work, not `https://`
```suggestion
**http://<ui_canister_id>.localhost:8000**
```